.github/workflows/code-quality.yml

```diff
@@ -71,6 +61,7 @@ jobs:
           --toolchain $HOME/conan/conan_paths.cmake
           -DCMAKE_CXX_STANDARD=17
           -DOPENASSETIO_WARNINGS_AS_ERRORS=ON
+          -DOPENASSETIO_ENABLE_CLANG_FORMAT=ON 
           -DOPENASSETIO_ENABLE_CLANG_TIDY=ON
           -DOPENASSETIO_ENABLE_CPPLINT=ON 
```

Comment:
[fix] Seems like we definitely need a CMake preset for this now, so we aren't relying on CI magic, and its easy to recreate locally.

Reply:
OK to convert this to a [plan], and I'll create an issue?

Reply:
Perfect, thanks @feltech

Reply:
Added #315